docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -42,6 +42,619 @@ For these cases, SNSs have so called 'generic proposals'. These are custom propo
+        ManageNervousSystemParameters : NervousSystemParameters;
+        AddGenericNervousSystemFunction : NervousSystemFunction;
+        RemoveGenericNervousSystemFunction : nat64;
+        UpgradeSnsToNextVersion : record {};
+        RegisterDappCanisters : RegisterDappCanisters;
+        TransferSnsTreasuryFunds : TransferSnsTreasuryFunds;
+        UpgradeSnsControlledCanister : UpgradeSnsControlledCanister;
+        DeregisterDappCanisters : DeregisterDappCanisters;
+        Unspecified : record {};
+        ManageSnsMetadata : ManageSnsMetadata;
+        ExecuteGenericNervousSystemFunction : ExecuteGenericNervousSystemFunction;
+        Motion : Motion;
+    };
+```
+
+## Using quill to submit proposals
+
+### Submitting via `sns make-proposal` command
+
+`sns make-proposal` signs a `ManageNeuron` message to [submit a proposal](https://github.com/dfinity/quill/blob/master/docs/cli-reference/sns/quill-sns-make-proposal.md). With this command, neuron holders can submit proposals (such as a Motion Proposal) to be voted on by other neuron holders. The structure:
+
+```bash
+quill sns make-proposal <PROPOSER_NEURON_ID> --proposal <PROPOSAL> [option]
+```
+
+where `<PROPOSAL>` is a formatted a candid record:
+
+```candid
+(
+    record {
+        title = "lorem ipsum";
+        url = "lorem ipsum";
+        summary = "lorem ipsum";
+        action = opt variant {
+            <PROPOSAL_TYPE> = <PARAMETERS_OF_PROPOSAL_TYPE>
+            }
+        };
+    }
+)
+```
+
+The CLI command structure is
+
+```bash
+quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
+    record {
+        title = "lorem ipsum";
+        url = "lorem ipsum";
+        summary = "lorem ipsum";
+        action = opt variant {
+            <PROPOSAL_TYPE> = <PARAMETERS_OF_PROPOSAL_TYPE>
+        };
+    }
+)'
+```
+
+For example, we use the candid record for <PROPOSAL_TYPE> `Motion`, the CLI-friendly command to submit a `Motion` proposal is:
+
+```bash
+quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
+    record {
+        title = "SNS Launch";
+        url = "https://dfinity.org\";
+        summary = "A motion to start the SNS";
+        action = opt variant {
+            Motion = record {
+                motion_text = "I hereby raise the motion that doesnt the use of the SNS shall commence";
+            }
+        };
+    }
+)'
+```
+
+## References for proposals
+
+### `Motion`
+
+### Relevant Type signature
+
+```candid
+    Motion: record {
+        motion_text : text;
+    }
+```
+
+### Putting it together
+
+Example in bash:
+
+```bash
+quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
+    record {
+        title = "lorem ipsum";
+        url = "lorem ipsum";
+        summary = "lorem ipsum";
+        action = opt variant {
+            Motion = record {
+                motion_text = "I hereby raise the motion that the use of the SNS shall commence";
+            }
+        };
+    }
+)'
+```
+
+## `ManageNervousSystemParameters`
+
```

Comment:
```suggestion
The nervous system parameters define the settings of a particular SNS. 
Since all SNSs are upgraded along the same upgrade path [refer to architecture part], this is what allows SNSs to nevertheless customise the rules of their governance, the tokenomics, etc to meet their needs. 

The proposal `ManageNervousSystemParameters` allows the SNS community to adjust these parameters at any time. 
```